README.md

```diff
@@ -83,6 +83,17 @@ Since each time the script is run the list of input files is shuffled, you don't
 workers will be processing the same video.
 On a rare occasion when the collision happens, the script will rewrite previously extracted features.
 
+
+## Input
+The inputs are paths to video files that are readable by OpenCV VideoReader.
```

Comment:
Actually, not all of them are readable by OpenCV VideoReader.

r21d and s3d rely on torchvision.io.read_video

Reply:
Or just removing it is fine, I'll patch it. The whole purpose of this PR is to have a description of what goes in and what goes out :)